### ISSUE TITLE
Use electron-builder for packaging

### ISSUE THREAD

**Issue by anderejd** [anderejd/electron-wasm-rust-example#7]
It seems to be the most mature and widely used option for packaging electron, based on github statistics.

https://github.com/electron-userland/electron-builder

**Comment by Boscop** [comment]
@anderejd Yes, please add an example way of packaging this into a standalone executable :)
I'm not sure what the best packager is, but [electron-forge](https://github.com/electron-userland/electron-forge) uses [electron-packager](https://github.com/electron/electron-packager) under the hood..

**Comment by anderejd** [comment]
If anyone wants to bite into this and send a PR that would be most welcome. This is just a small experiment on my part and the first time I tried using Electron for anything.
